support/mender-inventory-os

```diff
@@ -19,13 +19,15 @@ for file in /etc/os-release /usr/lib/os-release; do
     fi
 done
 
-if [ -x /usr/bin/lsb_release ]; then
-    OS="$(/usr/bin/lsb_release -sd)"
-    if [ -n "$OS" ]; then
-        echo "os=$OS"
-        exit 0
+for lsb_release in /bin/lsb_release /usr/bin/lsb_release; do
```

Comment:
Okey, but can't we just do something like: `if [ -x ``which lsb_release`` ]; then...` ?

Reply:
In principle I agree, but it's a bit tricky with the testing, which relies on which order these are attempted. I very much doubt this will be a problem on real devices, so I think I'll leave it like this for now to save time.

Reply:
Ok